views/beneficial-owner-gov.html

```diff
@@ -227,11 +244,12 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">Tell us about the government or pu
             text: "This is also known as the company or entity's governing law."
           },
           id: "corpLawGoverned",
-          name: "corpLawGoverned"
+          name: "corpLawGoverned",
```

Comment:
Are we sure about the abbreviations?

Reply:
i thought they were understandable but I am happy to expand them